src/cli/cli.cpp

```diff
@@ -968,6 +971,50 @@ void Interpreter::OutputNat64Counters(const otNat64Counters &aCounters)
 
 #endif // OPENTHREAD_CONFIG_NAT64_TRANSLATOR_ENABLE || OPENTHREAD_CONFIG_NAT64_BORDER_ROUTING_ENABLE
 
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_DHCP6_PD_ENABLE
+template <> otError Interpreter::Process<Cmd("pd")>(Arg aArgs[])
+{
+    otError error = OT_ERROR_NONE;
+
+    if (aArgs[0].IsEmpty())
+    {
+        ExitNow(error = OT_ERROR_INVALID_COMMAND);
+    }
+
+    /**
+     * @cli pd state
+     * @code
+     * pd state
+     * Disabled
+     * Stopped
+     * Running
+     * Done
+     * @endcode
+     * @par
+     * Gets the state of DHCPv6 PD feature.
+     *
+     */
+    if (aArgs[0] == "state")
+    {
+        static const char *const kDhcpv6PdState[] = {"Disabled", "Stopped", "Running"};
+        static_assert(0 == OT_BORDER_ROUTING_DHCP6_PD_STATE_DISABLED, "DHCPv6 PD is disabled on the border router");
```

Comment:
```suggestion
        static_assert(0 == OT_BORDER_ROUTING_DHCP6_PD_STATE_DISABLED, "The value of OT_BORDER_ROUTING_DHCP6_PD_STATE_DISABLED is not expected!");
```

Similar for following static_asserts.